### ISSUE TITLE
There's no indication of loading/progress

### ISSUE THREAD

**Issue by morgane** [nickbytes/nabu#2]
When I type in a URL and hit `enter`, I don't know if it's working or not until the page suddenly loads. Some feedback would be helpful here to indicate something *is* happening.

**Comment by nickbytes** [comment]
Spinner somewhere?

The words 'loading' on the page?

Some animation?

I'd be completely content with the words, "Loading..." and the three dots animating.

**Comment by morgane** [comment]
Yep, I think just something that says "Loading..." is fine, or a progress bar at the top of the page. Here's what such a progress bar could look like (blue line below URL bar)

![group 4](https://user-images.githubusercontent.com/104024/47297147-23c42500-d5c9-11e8-9b14-d255994c9681.png)
